di_website/templates/includes/cards/card-related.html

```diff
@@ -6,7 +6,8 @@
         <span class="card__tag">{{tag}}</span>
     {% endif %}
     {% image item.hero_image max-400x250 as hero_img %}
-    <span class="card__tag">{{ tag }}</span>
+    {% comment %} TODO: Use reusable partial for related content to load correct tags {% endcomment %}
+    <span class="card__tag">{{ item.content_type }}</span>
```

Comment:
Be careful when editing shared content ... has a larger impact.
In this case, we want to see a simple tag here... 'Blog', 'News', 'Event' e.t.c ... `content_type` doesn't fit.